app/api/services/audit.py

```diff
@@ -29,7 +29,7 @@ def log_audit_event(self, **kwargs):
 
 
 class AuditTypes(Enum):
-    update_price = 'update_price'
+    update_seller_price_admin = 'update_seller_price_admin'
```

Comment:
Can we keep the original name or was there a specific reason that we need to change it?